app/templates/main/index.html

```diff
@@ -139,6 +139,22 @@ <h2 class="ui dividing header">Information</h2>
   <div class="ui hidden divider"></div>
   <div class="actions">
     <h2 class="ui dividing header">Actions</h2>
+    <div class="text-save">
+        <div class="ui form">
+          <h3 class="ui header">Save this resource through SMS</h3>
+          <p class="disclaimer-sms">Standard messaging rates apply</p>
+          <div class="ui input">
+            <input type="text" id="phone-number" placeholder="1234567890">
+          </div>
+          <button class="ui button" id="text-save-submit">Text me</button>
+        </div>
+      </div>
+    </div>
+    <div class="textSentSuccess">
+      <div class="ui positive message">
+        Your text was sent!
+      </div>
+    </div>
     <div class="userRating" id="user-rating">
```

Comment:
maybe we should add a `ui hidden divider` between the SMS and ratings/reviews since rn there's no space between the text button and the Submit a Review header